src/store/transactions/exit/exit.thunks.ts

```diff
@@ -34,23 +34,21 @@ function fetchHermezAccount(
 
     dispatch(exitActions.loadAccount());
 
-    if (!wallet) {
-      return dispatch(exitActions.loadAccountFailure("Ethereum wallet is not loaded"));
-    }
-
-    return CoordinatorAPI.getAccount(accountIndex)
-      .then((account) =>
-        createAccount(
-          account,
-          poolTransactions,
-          undefined,
-          tokensPriceTask,
-          fiatExchangeRates,
-          preferredCurrency
+    if (wallet) {
```

Comment:
Is it me or after checking twice for `wallet !== undefined`, turns out we don't need the wallet?

The same problem is on `transfer.thunks.ts:fetchHermezAccount` where we're still dispatching the error `Ethereum wallet is not loaded`